src/source/parse_str.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const { parseScript } = require('shift-parser');
+const startJSONStream = require('./start-json-stream');
+
+startJSONStream(code => parseScript(code, { earlyErrors: false }));
```

Comment:
How does all of this interact with @arai-a's patch on WTF-8?

Reply:
This still uses his original `encodeWTF8` function as before, so it shouldn't change anything.

Reply:
Good!